docs/guides/gunw_product_guide.md

```diff
@@ -1,36 +1,84 @@
+You can download existing ARIA-S1-GUNW products from the Alaska Satellite Facility’s (ASF) 
+[Vertex](https://search.asf.alaska.edu/#/?dataset=SENTINEL-1%20INTERFEROGRAM%20(BETA) "Vertex ARIA-S1-GUNW Search" ){target=_blank} 
+search portal by following these steps: 
 
-1. **Access Vertex** – Go to the ASF Vertex website: [https://search.asf.alaska.edu](https://search.asf.alaska.edu){target=blank}.
-2. **Search for ARIA-S1-GUNW Products** – In the dataset selector, click on “ARIA S1 GUNW” to filter for these specific products. You can refine results by specifying a geographic region, date range, or other criteria using the search filters in the “filters” panel.
+1. **Access Vertex** – Go to the ASF Vertex website: 
+   [https://search.asf.alaska.edu](https://search.asf.alaska.edu "search.asf.alaska.edu" ){target=blank}.
+2. **Search for ARIA-S1-GUNW Products** – In the dataset selector, click on “ARIA S1 GUNW” to filter for these 
+   specific products. You can refine results by specifying a geographic region, date range, or other criteria 
+   using the search filters in the “filters” panel.
 ![Vertex ARIA S1 GUNW Dataset Selection](../images/vertex-GUNW-dataset-selection.png)
-3. **Preview and Select Products** – Click on individual results to view metadata, including coverage area and acquisition details.
-4. **Download Data** – To download, first add ARIA-S1-GUNW products to your download queue using the shopping cart icon next to each product, then download your selected products using the “download” panel.
+3. **Preview and Select Products** – Click on individual results to view metadata, including coverage area and 
+   acquisition details.
+4. **Download Data** – To download, first add ARIA-S1-GUNW products to your download queue using the shopping 
+   cart icon next to each product, then download your selected products using the “download” panel.
 
 ## Ordering On-Demand Products
 
-If the ARIA-S1-GUNW products you need are not available in the archive, you can use ASF's On Demand platform to submit custom ARIA-S1-GUNW jobs for processing. 
+If the ARIA-S1-GUNW products you need are not available in the archive, you can use ASF's On Demand platform to submit 
+custom ARIA-S1-GUNW jobs for processing. Once processing is complete, there are a couple of different approaches for 
+[accessing On Demand GUNW products](#accessing-on-demand-aria-s1-gunw-products "Jump to the Accessing On-Demand ARIA S1 GUNW Products section of this document"):
+
+- You can access them as you would any other 
+  [On-Demand](#accessing-products-using-on-demand-interfaces "Jump to the On Demand Interfaces section of this document" ) 
+  products from ASF.
+    - The download links provided will be active for 14 days.
+- Products generated On Demand are also added to the archive and can be accessed by 
```

Comment:
This statement once we offer customization for ARIA-S1-GUNW, but we'll need to make other changes at that time too so I think this is good for now.